project-tools.md

```diff
@@ -46,6 +46,7 @@ Name | Description
 [dotnet-nbench](https://github.com/petabridge/nbench) | A tool for running NBench performance tests [NuGet](https://www.nuget.org/packages/dotnet-nbench/)
 [dotnet-nswag](https://github.com/NSwag/NSwag) | Tools for NSwag, a Swagger 2.0 API for .NET, Web API, TypeScript. [NuGet](https://www.nuget.org/packages/NSwag.ConsoleCore/)
 [dotnet-outdated](https://github.com/goenning/dotnet-outdated) | DotNetOutdated is a tool to check for outdated .NET Core dependencies. [NuGet](https://www.nuget.org/packages/DotNetOutdated/)
+[dotnet-project-licenses](https://github.com/tomchavakis/nuget-license) | .NET Core tool to get the licenses of a project.[NuGet](https://www.nuget.org/packages/dotnet-project-licenses)
```

Comment:
Can you remove this entry? Your tool is not a project tool, so it cannot be used as a DotNetCliToolRefererence.

Reply:
I have removed this entry from project-tools.md